frontend/routes/signUp/signUp.component.tsx

```diff
@@ -170,6 +179,51 @@ export class SignUp extends React.PureComponent<IProps, IState> {
 			</MenuItem>
 		))
 
+	public renderIndustries = () =>
+		this.industries.map((industry) => (
+			<MenuItem key={industry} value={industry}>
+				{industry}
+			</MenuItem>
+		))
+
+	public renderHowDidYouFindUsOptions = () =>
+		this.howDidYouFindUsOptions.map((howDidYouFindUsOption) => (
+			<MenuItem key={howDidYouFindUsOption} value={howDidYouFindUsOption}>
+				{howDidYouFindUsOption}
+			</MenuItem>
+		))
+
+	public industries = [
```

Comment:
these are constants so they should really live in constants folder and get pulled in to use here.